### PR TITLE
tests: use playbook to setup the tested machine

### PR DESCRIPTION
**Pull request by praiskup**

### CODE REVIEW

File: mock/docs/release-instructions.txt [hunk 1]
```diff
@@ -24,11 +24,25 @@ Release checklist overview:
...
-3) run 'make check' in ./integration-tests directory and fix any reported
-   failures until it passes, you need to configure the box by ./setup-box.sh
-   first (don't run this on your box ideally, but temporarily allocated VM):
-   $ ./integration-tests/prepare/setup-box.sh
-   $ make check 1>/tmp/test-output.txt  2>&1
+3) run the testsuite, and fix any reported failures
+   It's strongly discouraged to run the testsuite locally, because certain
+   system configuration is needed.  Please allocate some disposable VM, and run
+   the testsuite there.  The system should be preconfigured using Ansible,
+   define the inventory (/etc/ansible/hosts):
```
Comment (xsuchy) [inline mock/docs/release-instructions.txt:31]: I prefer to have `ansible.cfg` in git root with:
```
[defaults]                                                                                                                                                                                   
inventory = ./inventory
```
And put this file in .gitignore

Comment (praiskup) [reply mock/docs/release-instructions.txt:31]: This though limits the directories where to run the playbook from :-/ otoh if you edit the global inventory you can run the playbook from anywhere...

Comment (praiskup) [reply mock/docs/release-instructions.txt:31]: OK, I will add a simple shell script, so users don't run ansible-playbook directly.

**Comment by praiskup** [comment]
$ grep -e FAIL -e PASS /tmp/test-output.txt  | grep -v DEBUG
*  PASSED: /home/mockbuild/mock/mock/integration-tests/001-orphanskill-explicit.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/002-copyin.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/01-shell.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/02-argpassing.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/03-chroot-one-arg.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/04-offline-tmpfs.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/05-orphanskill-std.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/06-retcode.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/07-init-clean.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/08-oldstyle-cmd.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/09-mockchain-fail.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/10-mockchain-partial.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/11-mockchain-success.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/12-mockchain-recurse.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/13-mockchain-gen-br.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/14-mock-multiple.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/15-overlayfs-layers.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/16-spec-and-multiple-srpms-fail.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/17-spec-and-rebuild.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/18-spec-sources-and-rebuild.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/19-sign-plugin.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/20-lvm-plugin.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/21-mockchain-image.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/22-rootdir.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/23-local-mirrorlist.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/24-postinstall.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/25-postupdate-root-cache.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/26-external-deps.tst
*  PASSED: /home/mockbuild/mock/mock/integration-tests/27-nspawn.tst
PASSED: alma+epel-8-x86_64.cfg (tmpfs)
PASSED: alma+epel-8-x86_64.cfg
PASSED: almalinux-8-x86_64.cfg (tmpfs)
PASSED: almalinux-8-x86_64.cfg
PASSED: centos-7-x86_64.cfg (tmpfs)
PASSED: centos-7-x86_64.cfg
PASSED: centos-8-x86_64.cfg (tmpfs)
PASSED: centos-8-x86_64.cfg
PASSED: centos+epel-7-x86_64.cfg (tmpfs)
PASSED: centos+epel-7-x86_64.cfg
PASSED: centos+epel-8-x86_64.cfg (tmpfs)
PASSED: centos+epel-8-x86_64.cfg
PASSED: centos-stream-8-x86_64.cfg (tmpfs)
PASSED: centos-stream-8-x86_64.cfg
PASSED: centos-stream-9-x86_64.cfg (tmpfs)
PASSED: centos-stream-9-x86_64.cfg
PASSED: centos-stream+epel-9-x86_64.cfg (tmpfs)
PASSED: centos-stream+epel-9-x86_64.cfg
PASSED: centos-stream+epel-next-8-x86_64.cfg (tmpfs)
PASSED: centos-stream+epel-next-8-x86_64.cfg
PASSED: centos-stream+epel-next-9-x86_64.cfg (tmpfs)
PASSED: centos-stream+epel-next-9-x86_64.cfg
PASSED: epel-7-x86_64.cfg (tmpfs)
PASSED: epel-7-x86_64.cfg
PASSED: eurolinux-8-i686.cfg (tmpfs)
PASSED: eurolinux-8-i686.cfg
PASSED: eurolinux-8-x86_64.cfg (tmpfs)
PASSED: eurolinux-8-x86_64.cfg
PASSED: fedora-34-i386.cfg (tmpfs)
PASSED: fedora-34-i386.cfg
PASSED: fedora-34-x86_64.cfg (tmpfs)
PASSED: fedora-34-x86_64.cfg
PASSED: fedora-35-i386.cfg (tmpfs)
PASSED: fedora-35-i386.cfg
PASSED: fedora-35-x86_64.cfg (tmpfs)
PASSED: fedora-35-x86_64.cfg
PASSED: fedora-36-i386.cfg (tmpfs)
PASSED: fedora-36-i386.cfg
PASSED: fedora-36-x86_64.cfg (tmpfs)
PASSED: fedora-36-x86_64.cfg
FAILED: fedora-eln-i386.cfg (tmpfs)
FAILED: fedora-eln-i386.cfg
PASSED: fedora-eln-x86_64.cfg (tmpfs)
PASSED: fedora-eln-x86_64.cfg
PASSED: fedora-rawhide-i386.cfg (tmpfs)
PASSED: fedora-rawhide-i386.cfg
PASSED: fedora-rawhide-x86_64.cfg (tmpfs)
PASSED: fedora-rawhide-x86_64.cfg
PASSED: mageia-7-i586.cfg (tmpfs)
PASSED: mageia-7-i586.cfg
PASSED: mageia-7-x86_64.cfg (tmpfs)
PASSED: mageia-7-x86_64.cfg
PASSED: mageia-8-i586.cfg (tmpfs)
PASSED: mageia-8-i586.cfg
PASSED: mageia-8-x86_64.cfg (tmpfs)
PASSED: mageia-8-x86_64.cfg
PASSED: mageia-9-i586.cfg (tmpfs)
PASSED: mageia-9-i586.cfg
PASSED: mageia-9-x86_64.cfg (tmpfs)
PASSED: mageia-9-x86_64.cfg
PASSED: mageia-cauldron-i586.cfg (tmpfs)
PASSED: mageia-cauldron-i586.cfg
PASSED: mageia-cauldron-x86_64.cfg (tmpfs)
PASSED: mageia-cauldron-x86_64.cfg
PASSED: navy-8-x86_64.cfg (tmpfs)
PASSED: navy-8-x86_64.cfg
PASSED: openmandriva-4.1-i686.cfg (tmpfs)
PASSED: openmandriva-4.1-i686.cfg
PASSED: openmandriva-4.1-x86_64.cfg (tmpfs)
PASSED: openmandriva-4.1-x86_64.cfg
PASSED: openmandriva-cooker-i686.cfg (tmpfs)
PASSED: openmandriva-cooker-i686.cfg
PASSED: openmandriva-cooker-x86_64.cfg (tmpfs)
PASSED: openmandriva-cooker-x86_64.cfg
PASSED: openmandriva-rolling-i686.cfg (tmpfs)
PASSED: openmandriva-rolling-i686.cfg
PASSED: openmandriva-rolling-x86_64.cfg (tmpfs)
PASSED: openmandriva-rolling-x86_64.cfg
PASSED: opensuse-leap-15.2-x86_64.cfg (tmpfs)
PASSED: opensuse-leap-15.2-x86_64.cfg
PASSED: opensuse-leap-15.3-x86_64.cfg (tmpfs)
PASSED: opensuse-leap-15.3-x86_64.cfg
PASSED: opensuse-tumbleweed-i586.cfg (tmpfs)
PASSED: opensuse-tumbleweed-i586.cfg
PASSED: opensuse-tumbleweed-x86_64.cfg (tmpfs)
PASSED: opensuse-tumbleweed-x86_64.cfg
PASSED: oraclelinux-7-x86_64.cfg (tmpfs)
PASSED: oraclelinux-7-x86_64.cfg
PASSED: oraclelinux-8-x86_64.cfg (tmpfs)
PASSED: oraclelinux-8-x86_64.cfg
FAILED: oraclelinux+epel-7-x86_64.cfg (tmpfs)
FAILED: oraclelinux+epel-7-x86_64.cfg
PASSED: oraclelinux+epel-8-x86_64.cfg (tmpfs)
PASSED: oraclelinux+epel-8-x86_64.cfg
PASSED: rhel-7-x86_64.cfg (tmpfs)
PASSED: rhel-7-x86_64.cfg
PASSED: rhel-8-x86_64.cfg (tmpfs)
PASSED: rhel-8-x86_64.cfg
PASSED: rhel+epel-8-x86_64.cfg (tmpfs)
PASSED: rhel+epel-8-x86_64.cfg
PASSED: rocky-8-x86_64.cfg (tmpfs)
PASSED: rocky-8-x86_64.cfg
PASSED: rocky+epel-8-x86_64.cfg (tmpfs)
PASSED: rocky+epel-8-x86_64.cfg


The ELN problem:
```
 Problem 1: cannot install the best candidate for the job
  - nothing provides go-srpm-macros needed by redhat-rpm-config-206-1.eln114.noarch
 Problem 2: package rpm-build-4.17.0-1.eln112.1.i686 requires system-rpm-config, but none of the providers can be installed
  - cannot install the best candidate for the job
  - nothing provides go-srpm-macros needed by redhat-rpm-config-206-1.eln114.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

The OracleLinux+epel 7 problem, should be fixed by the last commit
```
error: line 119: Unknown tag: Recommends: cvs
```

**Pull request merged**